src/Components/CreateImageWizardV2/CreateImageWizard.tsx

```diff
@@ -211,8 +212,10 @@ const CreateImageWizard = () => {
           <WizardStep
             name="Review"
             id="step-review"
-            footer={<CustomWizardFooter disableNext={true} />}
-          ></WizardStep>
+            footer={<CustomWizardFooter disableNext={false} />}
```

Comment:
This is very much a nit pick, but let's keep the Next disabled for now so we can't jump out of the Wizard this way.

```suggestion
            footer={<CustomWizardFooter disableNext={true} />}
```